apps/dashr-brain-viewer/app.R

```diff
@@ -0,0 +1,455 @@
+
+library(dashColorscales)
+library(dashR)
```

Comment:
Swithc to `dash` (and make sure app still runs with new versions of `CoreComponents` and `htmlComponents` sets)